### ISSUE TITLE
Is CMFA for initial authentication?

### ISSUE THREAD

**Issue by woodbe** [biometricITC/Administration#25]
With the difference iterations of the ESR, the question has been raised about whether CMFA is the initial authentication or if it takes over after initial authentication.

There are two "types" of authentication on a normal mobile device: the initial authentication (say after a reboot) and then a device unlock authentication. The main difference between these is in terms of key access, and not likely in anything that is directly CMFA, but it can be critical depending on what is expected for the use of CMFA.

On an initial authentication, the user keys are not yet unlocked, and so depending on how a vendor coded CMFA, it is possible that it is not yet available at this point (i.e. that it requires the user to login to access its settings). In this case, the CMFA cannot start until the device is initially started and authenticated by the user (that in this case we will trust because they had the right password only).

On device unlock, everything is running, and so there should not be any questions about availability as may be the case after a reboot. At device unlock, it would certainly seem that CMFA is likely running and in use, since the point of CMFA is whether or not the device should be locked in the first place.

But what is the expectation at the initial boot login? Should CMFA be active at this point, or should it accept that the entry of the password establishes trust and start after this event? If it is running at this point, how does it treat a password entry? I would expect this would be a trusted input, but what would we expect in terms of how it is handled (since it should allow access regardless of any other potential factors)?

As a related question, I would expect that the password entry would be an "alternative" authentication method (like in biometrics), so that CMFA failure (likely based on the user being unable to provide a correct biometric for some reason) should be able to be bypassed by the entry of the password to login. Is this what would be expected by everyone else?

**Comment by woodbe** [comment]
Points made in the discussion:

- the TOE could be an app instead of a built-in service, which could mean it has implementation considerations if the TOE must support the first authentication event
  - It was determined that this is acceptable and that the TOE must be able to work with all authentication events
- Can the user be blocked from logging in outside of certain conditions?
  - this isn't the intent for CMFA at this point, though it would seem possible to create such a configuration (though it could cause problems if those conditions can no longer be met)
- Could apps be blocked based on the CMFA conditions?
  - This could be done if the score info is available to a wider set, but would likely require apps to be aware as well
- Is a password sufficient to always pass the login?
  - likely, though it was pointed out that multiple factors could be required for a login which should be able to score higher (i.e. a password AND biometric for initial login)
- Password entry should be part of CMFA and not separate, so the TOE should be plugged in directly to the authentication system

**Comment by woodbe** [comment]
I don't know if this needs to be in the ESR. Maybe it needs to be in the PPM itself? I don't think this specifically has any bearing on the ESR though.

**Comment by woodbe** [comment]
I have modified the overall document to largely remove "sensor" in place of "input". There are still a lot of places where sensor makes sense, but I tweaked several sections to be more generic to "input" which could also be the password/PIN from the user as opposed to focusing solely on biometrics or other sensors. I think by adding that then this "solves" for CMFA at initial authentication or even those times when the user does enter in a password (for any reason).
